README.md

```diff
@@ -1,27 +1,51 @@
+The driver use case of SOMA is for single-cell data in the form of annotated matrices where observations are frequently cells and features are genes, proteins, or genomic regions.
 
-**1. SOMA.**
 
-SOMA, “stack of matrices, annotated,” is a flexible, extensible, and open-source API enabling access to data in a variety of formats. The vision for this API is that it will enable single cell datasets, including those with multiple modalities, to be stored in a cloud-friendly format and will be easily queryable, sliceable, and streamable without downloading or copying the full data. SOMA is designed to be general purpose and is grounded in the core assumption that the data can be modeled as a set of 2D annotated matrices that describe measurements of features across observations.
 
-The first implementation of the SOMA API is currently being built in partnership with [TileDB](https://tiledb.com) on top of their open-source (under the MIT License) [TileDB Embedded](https://tiledb.com/products/tiledb-embedded) storage engine. Both the Python and R APIs will be delivered incrementally in the upcoming months.
+## Motivation
 
-**2. Pilot project to offer CZ cellxgene’s Data Resource (30M+ cells and growing) via the SOMA API.**
+Datasets generated by profiling single cells are rapidly increasing in size and complexity. This has resulted in a need for scalable solutions to accommodate data sizes that no longer fit in memory and flexibility to accommodate the diversity of data being produced. 
 
-CZI is working on a pilot project to offer the entirety of cellxgene’s standardized single cell data resource, containing over 30 million cells, as a set of SOMA-backed objects. This resource, paired with the Python and R SOMA APIs bindings, will enable scientists to query, slice, and stream a subset of the data for analysis in downstream single cell toolkits.
+To address these emerging needs in the single cell ecosystem, CZI in partnership with TileDB is:
 
-Over the next few months, as the SOMA data model and API definition work mature, this resource will be offered to the public with accompanying notebooks demonstrating how to make use of the resource.
 
-**3. Supplementary domain specific libraries and schemas.**
+1. Driving the development of SOMA.
+2. Providing its first implementation, [TileDB-SOMA](https://github.com/single-cell-data/TileDB-SOMA) which utilizes the [TileDB Embedded](https://github.com/TileDB-Inc/TileDB) engine.
+3. Adopting TileDB-SOMA at [CZ CELLxGENE Discover](https://cellxgene.cziscience.com/) to build the [Cell Census](https://github.com/TileDB-Inc/TileDB) which provides efficient access and querying of its data from nearly 50 million cells, compiled from 700+ datasets.
```

Comment:
Cell Censusu has an incorrect link -- https://github.com/TileDB-Inc/TileDB points to our core engine

Reply:
cc: @pablo-gar in case you want to specify the landing page for that, if it's not going to be the repo (https://github.com/chanzuckerberg/cell-census/)

Reply:
yup good point, for now it is the repo so I will correct it.